app/src/main/java/is/xyz/mpv/MPVActivity.kt

```diff
@@ -997,16 +989,17 @@ class MPVActivity : AppCompatActivity(), MPVLib.EventObserver, TouchGesturesObse
             Log.e(TAG, "Failed to open content fd: $e")
             return null
```

Comment:
this branch could just return the original uri too

Reply:
Updated!

With ffmpeg supporting `content://` URIs natively now, is there still a benefit to calling `Utils.findRealPath()`? Would it make sense to always use the content URI?

Reply:
Real filesystem paths will still allow scripts (e.g. autoload.lua) to actually work with the filesystem, unlike content URIs.

Reply:
Ah, OK that makes sense.